keps/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -221,106 +218,34 @@ Metadata items:
 
 ### KEP Workflow
 
-TODO(jbeda) Rationalize this with status entires in the Metadata above.
-
-A KEP is proposed to have the following states
-
-- **opened**: a new KEP has been filed but not triaged by the responsible SIG or
-  working group
-- **accepted**: the motivation has been accepted by the SIG or working group as in
-  road map
-- **scoped**: the design has been approved by the SIG or working group
-- **started**: the implementation of the KEP has begun
-- **implemented**: the implementation of the KEP is complete
-- **deferred**: the KEP has been postponed by the SIG or working group despite
-  agreement on the motivation
-- **superseded**: the KEP has been superseded by another KEP
-- **retired**: the implementation of the KEP has been removed
-- **rejected**: the KEP has been rejected by the SIG or working group
-- **orphaned**: the author or developer of the KEP is no longer willing or able
-  to complete implementation
-
-with possible paths through the state space
-
-- opened -> deferred (a)
-- opened -> rejected (b)
-- opened -> orphaned (c)
-- opened -> accepted -> orphaned (d)
-- opened -> accepted -> scoped -> superseded (e)
-- opened -> accepted -> scoped -> orphaned (f)
-- opened -> accepted -> scoped -> started -> retired (g)
-- opened -> accepted -> scoped -> started -> orphaned (h)
-- opened -> accepted -> scoped -> started -> superseded (i)
-- opened -> accepted -> scoped -> started -> implemented (j)
-- opened -> accepted -> scoped -> started -> implemented -> retired (k)
-
-the happy path is denoted by (j) where an KEP is opened; accepted by a SIG as in
-their roadmap; fleshed out with a design; started; and finally implemented. As
-Kubernetes continues to mature, hopefully metrics on the utilization of features
-will drive decisions on what features to maintain and which to deprecate and so
-it is possible that a KEP would be retired if its functionality no longer provides
-sufficient value to the community.
+A KEP is has the following states
+
+- `Draft`: The KEP has been proposed and is actively being defined.
+- `Deferred`: The KEP is proposed but not actively being worked on.
+- `Approved`: Using lazy consensus, the approvers have approved this KEP.
+  It will move forward to implementation.
+- `Rejected`: The approvers and authors have decided that this KEP is not moving forward.
+  The KEP is kept around as a historical document.
+- `Withdrawn`: The KEP has been withdrawn by the authors.
+- `Final`: The KEP has been implemented and is no longer actively changed.
```

Comment:
My personal intention and perhaps also that of @jdumars was that these KEPs are living documents so that I don't think a `Final` state is needed at this time. Also whether or not a KEP is not being actively changed can be derived from either source control activity or the `mtime` on the KEP (which is probably what any automation would do anyway)

Reply:
I'd love to find a way to have folks sort through KEPs and find the ones that are still being actively implemented/worked on vs. those that are mostly frozen.  I'd like something that is easier to filter than just mtime.  Most of the static site tools and index tools don't have easy access to git history or mtime.

Reply:
I suppose my point is that automated tooling when it exists can add that sort of information when the site is (re)deployed and doesn't need to be mentioned here, at least not yet

Reply:
I'd like to keep things compatible with off the shelf tooling (static site generators) vs. having to invent a lot of new tooling.

Reply:
I'm trying to suggest that there might be other consumers besides static site generators so I wouldn't want to hamstring us up front. Also GitHub Markdown presentation is one of our _existing_ use cases so I think we should ensure whatever we do allows things to look nice here

Reply:
Not sure where the `Final` state impacts "looks nice" with github rendering. 

Github markdown presentation is very opinionated and not customizable.  Trying to optimize around that as this becomes a more comprehensive set of documents and want to make the presentation of those documents more and more useful.

Reply:
You're correct `Final` itself doesn't impact GitHub rendering and yes GitHub rendering is very opinionated; but it is documented and used in a variety of different rendering engines (or at least a compatible format). My point is that we _already_ have GitHub rendering as a reasonable design constraint since the KEPs are stored in a git repository hosted in GitHub. I don't want to speak too much for SIG Docs but they found it unworkable to deploy a static website directly from the branch where changes were being made (especially in order to support things like previews of a rendered site before publishing) so I think at a minimum a scalable automation story would have

1. contributor update KEPs
1. website automation pulls in changes
1. website automation lints KEPs and generates deployment artifacts and metadata (e.g. ToC, most commented KEPs, new KEPs, stale KEPs, recently changed KEPs)
1. website automation prepares preview for final human approval
1. website automation deploys new website

so I don't think it's required to bundle deployment specific (e.g. Hugo, GitHub Pages) metadata within the KEPs at least not until we have pretty wide adoption of any of the tooling we build on top of flat files checked into git

Reply:
One of the lessons from SIG Docs was the usage of netlify to help manage deployment.  This is something that we are doing as we build this site out.